File: app/src/main/java/com/seungjun/randomox/network/data/UserInfo.java
package com.seungjun.randomox.network.data;

import com.google.gson.annotations.SerializedName;

public class UserInfo extends HeaderInfo {

    @SerializedName("user_key")
    public String user_key = "";

    @SerializedName("user_point")
    public int user_point = 0;

    @SerializedName("user_sIndex")
    public int user_sIndex = 0;
}

File: app/src/main/java/com/seungjun/randomox/network/data/UserInfo.kt
package com.seungjun.randomox.network.data

import com.google.gson.annotations.SerializedName

data class UserInfo(@SerializedName("user_key") var user_key: String = "",
                    @SerializedName("user_point") var user_point: Int = 0,
                    @SerializedName("user_sIndex") var user_sIndex: Int = 0,
                    @SerializedName("rank") var rank: Int = 0) : HeaderInfo()
